test/tests-harmony.js

```diff
@@ -3132,75 +3132,6 @@ test("[a, b] = [b, a]", {
   locations: true
 });
 
-test("({ responseText: text }) = res", {
```

Comment:
Changing this to a destructuring assignment without parens is already tested below (var {a:b} = {})